prometheus/build.gradle

```diff
@@ -42,7 +42,7 @@ test {
 configurations.all {
     resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.6.0"
     resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0"
-    resolutionStrategy.force "org.junit.jupiter:junit-jupiter:5.6.2"
+    resolutionStrategy.force 'junit:junit:4.13.2'
```

Comment:
why is this downgrading?

Reply:
https://mvnrepository.com/artifact/junit/junit
Its the latest one.